LLama/LLamaExecutorBase.cs

```diff
@@ -315,6 +315,34 @@ public virtual async IAsyncEnumerable<string> InferAsync(string text, IInference
             }
         }
 
+        /// <summary>
+        /// Asynchronously runs a prompt through the model to compute KV cache without generating any new tokens.
+        /// </summary>
+        /// <param name="prompt">Prompt to process</param>
+        /// <param name="cancellationToken">A cancellation token</param>
+        /// <returns></returns>
+        public virtual async Task AddPromptAsync(string prompt, CancellationToken cancellationToken = default)
```

Comment:
```suggestion
        public virtual async Task PrefillPromptAsync(string prompt)
```

Please remove the `canlellationToken` since it's not used and `prefill` matches the functionality of the method better.